modules/gapi/test/infer/gapi_infer_ie_test.cpp

```diff
@@ -2028,6 +2039,24 @@ TEST_F(ROIList, CallInferMultipleTimes)
     validate();
 }
 
+TEST(IEFrameAdapter, blobParams)
+{
+    cv::Mat bgr = cv::Mat::eye(240, 320, CV_8UC3);
+    cv::MediaFrame frame = cv::MediaFrame::Create<TestMediaBGR>(bgr);
+
+    cv::util::any any_params = frame.blobParams();
```

Comment:
Why not just `params`

Reply:
Ok, but you don't need to store it in separate variable

Reply:
Done